manuscript/recipes/duplicati.md

```diff
@@ -112,4 +104,16 @@ networks:
 
 Launch the Duplicati stack by running ```docker stack deploy duplicati -c <path-to-docker-compose.yml>```
 
-Authenticate against your OAuth provider, and then start configuring your backup jobs via the Duplicati UI!
\ No newline at end of file
+### Create (and verify!) Your First Backup
+Once we authenticate through the traefik-forward-auth provider, we can start configuring your backup jobs via the Duplicati UI. All backup and restore job configuration is done through the UI. Be sure to read through the documentation on [Creating a new backup job](https://duplicati.readthedocs.io/en/latest/03-using-the-graphical-user-interface/#creating-a-new-backup-job) and [Restoring files from a backup](https://duplicati.readthedocs.io/en/latest/03-using-the-graphical-user-interface/#restoring-files-from-a-backup) for information on how to configure those jobs. 
+
+!!! warning
+    An untested backup is not really a backup at all. Being ***sure*** you can succesfully restore files from your backup now could save you lots of heartache later after "something bad" happens.
+
+!!! tip
+    Backing up files on a regular basis is going to use a continually-increasing amount of disk space. To help with this, Duplicati offers a "Smart Backup Retention" scheme that will intelligently remove certain backups as they age while still maintaining a comprehensive backup history. You can set that configuration on the "Options" tab of the backup configuration.
+
+## Chef's Notes 📓
+
+[^1]: The [Duplicati 2 User's Manual](https://duplicati.readthedocs.io/en/latest/) contains all the information you'll need to configure backup endpoints, restore jobs, scheduling and advanced properties for your backup jobs.
```

Comment:
I _think_ this is getting formatted strangely (after the comments section) because of the whitespace preceding it. Try removing the whitespace?

Reply:
Fixed